labgrid/remote/client.py

```diff
@@ -1795,6 +1794,15 @@ def main():
         try:
             signal.signal(signal.SIGTERM, lambda *_: sys.exit(0))
 
+            if args.crossbar is None and env is not None:
+                # If no crossbar URL provided, try config file crossbar_url
+                args.crossbar = env.config.get_option('crossbar_url', None)
+
+            if args.crossbar is None:
+                # then LG_CROSSBAR environment variable
+                # fallback to default
+                args.crossbar = os.environ.get("LG_CROSSBAR", "ws://127.0.0.1:20408/ws")
```

Comment:
The `crossbar_realm` option should also be read from the environment configuration.

Reply:
I've added reading the realm from the configuration file.